assets/dev/js/editor/regions/navigator/elements/hooks/ui/panel/editor/open/navigator-toggle-list.js

```diff
@@ -0,0 +1,34 @@
+import After from 'elementor-api/modules/hooks/ui/after';
+
+export class NavigatorToggleList extends After {
```

Comment:
Explain + check if the navigator is open.

Reply:
Handled: https://github.com/elementor/elementor/pull/14264/commits/b2be23d45787ac398a1243da5b24230a36c44e0e